### ISSUE TITLE
Dutch F3 dataset not usin "val" split - uses train set, using a separate validation set may improve the model performance

### ISSUE THREAD

**Issue by maxkazmsft** [microsoft/seismic-deeplearning#143]
Fix Dutch F3 loader to use the validation split away from the training set

**Comment by yalaudah** [comment]
This is related to #233 and #200 .
